about.html

```diff
@@ -0,0 +1,161 @@
+relative_css_url: /assets/css/about.css
+title: IT Staffing Firm, Remote Development Teams | Active Bridge
+description: We are a contract development agency specializing in providing
+              on-demand full-stack engineers to help our clients and augment
+              their in-house teams.
+---
+
+<div class="container about" id="container">
+  <div class="info-container wrap">
+    <div class="row">
+      <h1 class="header-3 white-text">{{ site.data.about.about }}</h1>
+      <h1 class="about__header">{{ site.data.about.header }}</h1>
+      <div class="about__description">{{ site.data.about.description }}</div>
+      <div class="about_us_info">
+        <div class="about_us_info__items">
+          <div class="about_us_info__container">
+            {% for item in site.data.index.about_us_items %}
+              <div class="about_us_info__item white-text">
+                <img src="/assets/images/index/{{ item.logo }}" alt="Logo">
+                {{ item.text }}
+              </div>
+            {% endfor %}
+          </div>
+          <div class="about_us_info__photo desktop-only">
+            <img src="/assets/images/about/about-bg-2.webp" alt="About us">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="about__vision wrap">
+    <div class="row">
+      <div class="about__vision-description about__description">
+        {{ site.data.about.information.first }}
+      </div>
+      <div class="about__vision-description about__description">
+        {{ site.data.about.information.second }}
+      </div>
+      <h2 class="header-3">{{ site.data.about.company_vision.titlet }}</h2>
+      <h2 class="header-2">{{ site.data.about.company_vision.subtitle }}</h2>
+      <div class="about__vision-description about__description about__description-cera">
+        {{ site.data.about.company_vision.description }}
+      </div>
+      <div class="chart-container">
+        {% include expertise_chart.html %}
+        <a class="button" href="/expertise" id="about-chart-container-button">{{ site.data.about.company_vision.button }}</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="about__crew wrap">
+    <div class="row">
+      <div class="about__description">{{ site.data.about.crew.description }}</div>
+      <div class="header-3 white-text">{{ site.data.about.crew.title }}</div>
+      {% include crew.html %}
+      <a class="button about__crew-button" id="about__crew-button" href="/about/team">
+        {{ site.data.about.crew.button }}
+      </a>
+    </div>
+  </div>
+
+  <div class="about__benefits wrap">
+    <div class="overlay"></div>
+    <div class="row">
+      <div class="header-3">{{ site.data.about.benefits.title }}</div>
+      <h2 class="header-2">{{ site.data.about.benefits.subtitle }}</h2>
+      <div class="about__vision-descriptionabout__description about__description-cera">
+        {{ site.data.about.benefits.description }}
+      </div>
+      <div class="accordion">
```

Comment:
The text above this `.accordion` section is blurred with the closed state.
the down arrow is not center aligned.
I can't click on a section to expand, only on the text

https://github.com/katatsu12/live_blog/assets/22080708/149acdb3-ebfc-4dc0-8a1d-d3f026fc3577